CPAC/anat_preproc/utils.py

```diff
@@ -3,6 +3,30 @@
 
 from CPAC.pipeline import nipype_pipeline_engine as pe
 
+def pad(cropped_image_path, target_image_path):
+    from numpy import asanyarray, zeros_like
```

Comment:
Can you [add a docstring](https://github.com/FCP-INDI/.github/blob/main/CONTRIBUTING.md#code-commenting) for this function? I can tell from the PR what is happening here, but if I was just looking in the code, I think I'd need to find where it's called to know what it does

Reply:
Thanks for comment. Added now